src/google.cr

```diff
@@ -58,7 +58,12 @@ module PlaceCalendar
 
     def list_calendars(mail : String, **options) : Array(Calendar)
       if calendars = calendar(mail).calendar_list
-        calendars.map { |c| c.to_place_calendar }
+        # filtering out hidden and rejected calendars as seen in google-staff-api
+        calendars.reject { |item|
+          item.hidden || item.deleted
+        }.map { |item|
+          item.to_place_calendar
+        }
```

Comment:
This can be done in a single iteration

```suggestion
        calendars.compact_map { |item|
          item.to_place_calendar unless item.hidden || item.deleted
        }
```

Reply:
👍